CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Thu Mar 18 2021 Steven Pritchard <steven.pritchard@onyxpoint.com> - 3.1.4-0
```

Comment:
Can you please combine these into a single changelog entry since they're the same author and release?

Reply:
Done.